lib/booktypecontrol/forms.py

```diff
@@ -140,20 +140,10 @@ def save_settings(self, request):
 
 
 class FrontpageForm(BaseControlForm, forms.Form):
-    description = forms.CharField(
```

Comment:
Hi Helmy, this has to do with class FrontPageView on line 42 of https://github.com/sourcefabric/Booktype/blob/master/lib/booktype/apps/portal/views.py

If you look at the controls for the site front page, for example http://booktype-dev.sourcefabric.org/_control/settings/#frontpage you will see the legacy fields 'Welcome message' and 'Show activity' which were used by a previous design of the front page.

Currently, any content you put in the welcome message field is not actually used by the front page template and no activity is shown even if you check this box and uncheck the anonymous page box.

Also the help text for 'Use anonymous page' is out of date because the anonymous page is the default now.